Tools/AP_Periph/rangefinder.cpp

```diff
@@ -31,67 +34,92 @@ void AP_Periph_FW::can_rangefinder_update(void)
     }
 #endif
     uint32_t now = AP_HAL::millis();
-    static uint32_t last_update_ms;
     if (g.rangefinder_max_rate > 0 &&
-        now - last_update_ms < uint32_t(1000/g.rangefinder_max_rate)) {
-        // limit to max rate
+        now - last_rangefinder_update_ms < uint32_t(1000/(g.rangefinder_max_rate*rangefinder.num_sensors()))) {
```

Comment:
don't divide by number of sensors, send all at the desired rate

Reply:
Done